src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -402,6 +412,7 @@ const NetworkModificationNodeEditor = ({ selectedNode }) => {
                 className={classes.addButton}
                 color="primary"
                 onClick={openNetworkModificationConfiguration}
+                disabled={oneNodeBuilding}
             >
                 <AddIcon />
             </Fab>
```

Comment:
Maybe disable drag&drop also when one node is building ?

Reply:
Done. I also dissabled edit button.